thedevopser/castor-symfony/1.0/manifest.json

```
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "TheDevOpser\\CastorBundle\\CastorBundle": [
+            "all"
+        ]
```

It's more common to write it like this

```suggestion
        "TheDevOpser\\CastorBundle\\CastorBundle": ["all"]
```

---

Thank you for the return, I have made the modification.